release/staging-build.yml

```diff
@@ -134,6 +134,61 @@ batch:
         variables:
           PROJECT_PATH: projects/aws/rolesanywhere-credential-helper
           CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/aws.rolesanywhere-credential-helper
+    - identifier: aws_upgrader_1_24
```

Comment:
Na we dont need these during checksums. Add the ignore_from_checksum_build thing, look at the packages projects for the exact name.